defaults.js

```diff
@@ -27,7 +27,11 @@ module.exports = function setDefaults (name, config) {
       hops: 3
     },
     gossip: {
-      connections: 3
+      connections: 3,
```

Comment:
TODO - remove connections, it seems to not be used anywhere!

Reply:
we could make it work